src/module.c

```diff
@@ -8805,6 +8805,71 @@ int RM_ACLAddLogEntry(RedisModuleCtx *ctx, RedisModuleUser *user, RedisModuleStr
     return REDISMODULE_OK;
 }
 
+/* This is to prevent a coupling between redis internals and modules */
+uint64_t mapInternalScriptFlagsToModuleFlags(uint64_t flags) {
```

Comment:
I'm always having hard time to find a style while creating a PR for redis but I guess for "if checks", it is single line or no braces generally, just fyi

https://github.com/redis/redis/blob/eacca729a55501508c434bab30c2432e58728aee/src/module.c#L8359
https://github.com/redis/redis/blob/eacca729a55501508c434bab30c2432e58728aee/src/module.c#L11626